6/changelog.yaml

```diff
@@ -86,3 +86,11 @@ releases:
 
         `Porting Guide <https://docs.ansible.com/ansible/devel/porting_guides.html>`_'
     release_date: '2022-08-02'
+  6.3.0:
+    changes:
+      deprecated_features:
+      - The google.cloud collection is considered unmaintained and will be
+        removed from Ansible 8 if nobody starts maintaining it again until then.
+        See `the removal process for details on how this works
+        <https://github.com/ansible-collections/overview/blob/main/removal_from_ansible.rst#cancelling-removal-of-an-unmaintained-collection>`__
```

Comment:
I hadn't seen the double underscore before and wanted to know it was a typo, but it is not! TIL
https://stackoverflow.com/a/27435441/3905079

Reply:
omgosh ditto!  TIL... and I think I've seen some of those duplicate target name warnings in the docs build and never figured out what was wrong... :-)

Reply:
I'm expecting that in the future we'll copy'n'paste this fragment, so the double underscore will prevent that warning then :-)